src/constraints.jl

```diff
@@ -239,3 +239,98 @@ function set_coefficient(constraint::ConstraintRef{Model, MOICON{F, S}},
         MOI.ScalarCoefficientChange(index(variable), convert(T, value)))
     return
 end
+
+"""
+    shadow_price(constraint::ConstraintRef)
+
+The change in the objective from an infinitesimal relaxation of the constraint.
+This value is computed from [`result_dual`](@ref) and can be queried only when
+`has_result_dual` is `true` and the objective sense is `MinSense` or `MaxSense`.
```

Comment:
Most user will not understand what the objective sense can be if it is not `MinSense` nor `MaxSense`.